### PR TITLE
[FLINK-19394][docs-zh] Translate the 'Monitoring Checkpointing' page of 'Debugging & Monitoring' into Chinese

### PR DESCRIPTION
**Pull request by RocMarshal**
## What is the purpose of the change

*Translate the 'Monitoring Checkpointing' page of 'Debugging & Monitoring' into Chinese*

## Brief change log

*Translate the 'Monitoring Checkpointing' page of 'Debugging & Monitoring' into Chinese*
  - *page file location: docs/monitoring/checkpoint_monitoring.zh.md*
  - *page url in documentation-web is : https://ci.apache.org/projects/flink/flink-docs-release-1.11/zh/monitoring/checkpoint_monitoring.html*

## Verifying this change

*A pure translation work .*

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / **no**)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / **no**)
  - The serializers: (yes / **no** / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / **no** / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: (yes / **no** / don't know)
  - The S3 file system connector: (yes / **no** / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / **no**)
  - If yes, how is the feature documented? (**not applicable** / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by RocMarshal** [comment]
Hi, @klion26
Could you help me to review this PR if you have free time?
Thank you.

File: docs/monitoring/checkpoint_monitoring.zh.md [hunk 1]
```diff
@@ -25,105 +25,125 @@ under the License.
...
+- **ID**：已触发 Checkpoint 的 ID。每个 Checkpoint 的 ID 都会递增，从 1 开始。
+- **Status**：Checkpoint 的当前状态，可以是*正在进行（In Progress）*（<i aria-hidden="true" class="fa fa-circle-o-notch fa-spin fa-fw"/>）、*已完成（Completed）*（<i aria-hidden="true" class="fa fa-check"/>）或*失败（Failed）*（<i aria-hidden="true" class="fa fa-remove"/>）。如果触发的检查点是一个保存点，你将看到一个 <i aria-hidden="true" class="fa fa-floppy-o"/> 符号。
+- **Trigger Time**：在 JobManager 上发起 Checkpoint 的时间。
+- **Latest Acknowledgement**：JobManager 接收到任何 subtask 的最新确认的时间（如果尚未收到确认，则不适用）。
+- **End to End Duration**：从触发时间戳到最后一次确认的持续时间（如果还没有收到确认，则不适用）。完整 Checkpoint 的端到端持续时间由确认 Checkpoint 的最后一个 Subtask 确定。这个时间通常大于单个 Subtask 实际 Checkpoint State 所需的时间。
+- **Checkpointed Data Size**：所有已确认 Subtasks 的 checkpoint 的数据大小。如果启用了增量 Checkpoint，则此值为 Checkpoint 数据的增量大小。
+- **Processed in-flight data**：在对齐（Checkpoint alignment）期间（从接收第一个和最后一个 Checkpoint barrier 之间的时间）对所有已确认的 Subtasks 处理的大约字节数。
+- **Persisted in-flight data**：在对齐期间（从接收第一个和最后一个 Checkpoint barrier 之间的时间）对所有已确认的 Subtasks 持久化的字节数。仅当启用未对齐的 Checkpoint 时，此值大于 0。
```
Comment (klion26) [inline docs/monitoring/checkpoint_monitoring.zh.md:74]: unaligned checkpoint 翻译成 `未对齐的 Checkpoint` 这个是否合适呢？

File: docs/monitoring/checkpoint_monitoring.zh.md [hunk 2]
```diff
@@ -25,105 +25,125 @@ under the License.
...
-- **Alignment Duration**: The time between processing the first and the last checkpoint barrier. For aligned checkpoints, during the alignment, the channels that have already received checkpoint barrier are blocked from processing more data.
-- **Start Delay**: The time it took for the first checkpoint barrier to reach this subtasks since the checkpoint barrier has been created.
+- **Sync Duration**：Checkpoint 同步部分的持续时间。这包括 operators 的快照状态，并阻塞 subtask 上的所有其他活动（处理记录、触发计时器等）。
+- **Async Duration**：Checkpoint 的异步部分的持续时间。这包括将 Checkpoint 写入设置的文件系统所需的时间。对于未对齐的 Checkpoints，这还包括 subtask 必须等待最后一个 checkpoint barrier 到达的时间（对齐持续时间）以及持久化数据所需的时间。
+- **Alignment Duration**：处理第一个和最后一个 checkpoint barrier 之间的时间。对于对齐机制的 checkpoints，在对齐过程中，已经接收到 checkpoint barrier 的 channels 将阻塞并停止处理后续的数据。
+- **Start Delay**：自创建 checkpoint barrier 开始，第一个 checkpoint barrier 到达此 subtask 所用的时间。
```
Comment (klion26) [inline docs/monitoring/checkpoint_monitoring.zh.md:85]: 这个的意思能否再优化下呢？如果我没猜错的话，这里应该是 "从 barrier 在 source 创建到流到这个 task 的时间差“ 当然这个含义我需要看代码 check 下

File: docs/monitoring/checkpoint_monitoring.zh.md [hunk 3]
```diff
@@ -25,105 +25,125 @@ under the License.
...
 web.checkpoints.history: 15
 {% endhighlight %}
 
-### Summary Tab
+<a name="summary-tab"></a>
+
+### 摘要信息（Summary）选项卡
 
-The summary computes a simple min/average/maximum statistics over all completed checkpoints for the End to End Duration, Checkpointed Data Size, and Bytes Buffered During Alignment (see [History](#history) for details about what these mean).
+摘要计算了所有已完成 checkpoints 的端到端持续时间、Checkpointed 数据大小和对齐期间缓冲的字节数的简单 min/average/maximum 统计信息（有关这些内容的详细信息，请参见 [History](#History tab)）。
```
Comment (klion26) [inline docs/monitoring/checkpoint_monitoring.zh.md:102]: 这个 history 的 tab 链接应该是有问题的

File: docs/monitoring/checkpoint_monitoring.zh.md [hunk 4]
```diff
@@ -25,105 +25,125 @@ under the License.
...
 
 <center>
-  <img src="{{ site.baseurl }}/fig/checkpoint_monitoring-summary.png" width="700px" alt="Checkpoint Monitoring: Summary">
+  <img src="{% link /fig/checkpoint_monitoring-summary.png %}" width="700px" alt="Checkpoint Monitoring: Summary">
 </center>
 
-Note that these statistics don't survive a JobManager loss and are reset to if your JobManager fails over.
+请注意，这些统计信息不会在 JobManager 丢失后无法保存，如果 JobManager 故障转移，这些统计信息将被重置。
```
Comment (klion26) [inline docs/monitoring/checkpoint_monitoring.zh.md:108]: 这里的意思应该是 jm 重启后，这些信息会丢失，和上文说的一样，会重新计数

File: docs/monitoring/checkpoint_monitoring.zh.md [hunk 5]
```diff
@@ -25,105 +25,125 @@ under the License.
...
+
+<a name="configuration-tab"></a>
 
-### Configuration Tab
+### 配置信息（Configuration）选项卡
 
-The configuration list your streaming configuration:
+该配置选项卡列出了你指定的流式配置：
```
Comment (klion26) [inline docs/monitoring/checkpoint_monitoring.zh.md:114]: streaming configuration 翻译成 `流式配置` 不确定是否合适

File: docs/monitoring/checkpoint_monitoring.zh.md [hunk 6]
```diff
@@ -25,105 +25,125 @@ under the License.
...
-- **Timeout**: Timeout after which a checkpoint is cancelled by the JobManager and a new checkpoint is triggered.
-- **Minimum Pause Between Checkpoints**: Minimum required pause between checkpoints. After a checkpoint has completed successfully, we wait at least for this amount of time before triggering the next one, potentially delaying the regular interval.
-- **Maximum Concurrent Checkpoints**: The maximum number of checkpoints that can be in progress concurrently.
-- **Persist Checkpoints Externally**: Enabled or Disabled. If enabled, furthermore lists the cleanup config for externalized checkpoints (delete or retain on cancellation).
+- **Checkpointing Mode**：*恰好一次（Exactly Once）*或者*至少一次（At least Once）*。
+- **Interval**：配置的 Checkpoint 触发间隔。在此间隔内触发 Checkpoint。
+- **Timeout**：超时之后，JobManager 取消 checkpoint 并触发新的 checkpoint。
+- **Minimum Pause Between Checkpoints**：Checkpoint 之间所需的最小暂停时间。Checkpoint 成功完成后，我们至少要等这段时间再触发下一个，这可能会延迟正常的间隔。
+- **Maximum Concurrent Checkpoints**：可以同时进行的最大 checkpoint 个数。
+- **Persist Checkpoints Externally**：启用或禁用持久化 Checkpoints 到外部系统。如果启用，还会列出外部化 checkpoints 的清理配置（取消时删除或保留）。
```
Comment (klion26) [inline docs/monitoring/checkpoint_monitoring.zh.md:121]: `Checkpoint` 的大小写建议统一，非首字母可以小写

**Review by klion26 — COMMENTED** [review]
@RocMarshal thanks for the contribution, overall the change looks good, left some minor comments. please take a look.

**Comment by RocMarshal** [comment]
@klion26 
Thank you  for your help.
I made some change based  on  your suggestions.

**Comment by RocMarshal** [comment]
@xccui

**Review by klion26 — APPROVED** [review]
@RocMarshal thanks for the update, LGTM now.
cc @wuchong  do you want to have a double-check this

**Comment by wuchong** [comment]
Feel free to merge @kl0u .

**Comment by RocMarshal** [comment]
@dianfu 
Could you help me to merge this PR if there is nothing inappropriate ?

Thank you.

**Comment by dianfu** [comment]
@RocMarshal I'd like to leave it to @klion26 who is more professional in this area and have also reviewed this PR. @klion26 Feel free to merge this PR if you think it's OK to merge. :)

**Comment by klion26** [comment]
@dianfu thanks for the review, will merge this one soon.

**Comment by klion26** [comment]
@RocMarshal When I pull your branch locally, seems there are two other commits between 296c869ceac85 and 9793b9c23786a9, I'm not sure why could this happen, could you please double-check it on your side? 

And very much appreciate it if you could squash the commit into a single one, I'm not very familiar with merge commit into master currently :)


```
275378338d (HEAD -> RE_13750, R13750) Update docs/monitoring/checkpoint_monitoring.zh.md
1ec411ddcf Update docs/monitoring/checkpoint_monitoring.zh.md
9793b9c237 Update docs/monitoring/checkpoint_monitoring.zh.md
726435e10d [FLINK-19394][docs-zh] Translate the 'Monitoring Checkpointing' page of 'Debugging & Monitoring' into Chinese
3b9e20aafc [FLINK-17528][table] Remove ArrayData#get() API and use ElementGetter instead
debbdc66a4 [FLINK-17528][table] Remove RowData#get() API and use FieldGetter instead
296c869cea [FLINK-19394][docs-zh] Translate the 'Monitoring Checkpointing' page of 'Debugging & Monitoring' into Chinese
```

**Comment by RocMarshal** [comment]
> @RocMarshal When I pull your branch locally, seems there are two other commits between [296c869](https://github.com/apache/flink/commit/296c869ceac85a5fd5c880a371f3bc6941f97acb) and [9793b9c](https://github.com/apache/flink/commit/9793b9c23786a99b65dbe044f84fb5d1232add20), I'm not sure why could this happen, could you please double-check it on your side?
> 
> And very much appreciate it if you could squash the commit into a single one, I'm not very familiar with merge commit into master currently :)
> 
> ```
> 275378338d (HEAD -> RE_13750, R13750) Update docs/monitoring/checkpoint_monitoring.zh.md
> 1ec411ddcf Update docs/monitoring/checkpoint_monitoring.zh.md
> 9793b9c237 Update docs/monitoring/checkpoint_monitoring.zh.md
> 726435e10d [FLINK-19394][docs-zh] Translate the 'Monitoring Checkpointing' page of 'Debugging & Monitoring' into Chinese
> 3b9e20aafc [FLINK-17528][table] Remove ArrayData#get() API and use ElementGetter instead
> debbdc66a4 [FLINK-17528][table] Remove RowData#get() API and use FieldGetter instead
> 296c869cea [FLINK-19394][docs-zh] Translate the 'Monitoring Checkpointing' page of 'Debugging & Monitoring' into Chinese
> ```

@klion26 I'll check it as soon as possible. Thank you for your reply.

**Comment by RocMarshal** [comment]
@klion26 I made a little change on the branch information. Please take a look.

**Comment by klion26** [comment]
@RocMarshal thanks for the update, merging.

**Pull request merged**